doc/1.4/language.md

```diff
@@ -609,10 +609,15 @@ attribute `a`, then one can access it as follows:
 log info: "the value of attribute a is %d", dev.a.val;
 ```
 
-The [`init` template](dml-builtins.html#init) is often useful together with
-`attribute` objects &mdash; in particular in combination with these standard
-templates &mdash; in order to initialize the state associated with the
-`attribute` object.
+These templates also provide an `init_val` parameter and an overridable
+implementation of [`init`](dml-builtins.html#init) that leverages said
+parameter to initialize the `val` session variable (with the default
+being zero-initalizising the variable.)
```

Comment:
the `init` _method_.

Heavy language, maybe "The default behaviour of the `init` method is to initialize... The default is to .."

Reply:
Maybe... I don't like the repetition of saying "The default behaviour...", "The default is to...". That's why I said "overridable". I'm not sure I agree this is heavy language, when reading it I feel it flows decently well.

Reply:
Re. referencing init: can I get away with saying "an overridable implementation of `init()` ..."?

Reply:
It's generally easier to read you keep sentences shorter; in this case you say 3 fairly free-standing things so good to split in 3 sentences.

Yes, I like the `init()` suggestion